doc/source/operations/dell-firmware-update.rst

```diff
@@ -0,0 +1,161 @@
+===============================
+Dell Firmware Update Automation
+===============================
+
+Overview
+========
+
+Custom playbooks are available to automate firmware updates on Dell hardware.
+
+We make use of `Dell Repository Manager (DRM)
+<https://www.dell.com/support/kbdoc/en-uk/000177083/support-for-dell-emc-repository-manager-drm>`__.
+
+Prerequisites
+=============
+
+DRM needs to listen on port 443 and needs access to the out-of-band management
+network. Choose a host where it won't conflict with another service.
+
+To run DRM in a container, first start a container, that has a Docker volume to
+host all the firmware files:
+
+.. code-block:: bash
+
+   docker volume create dell_firmware
+   docker run --detach -v dell_firmware:/dell_firmware --name dell-drm --network host --restart always rockylinux:9.3 sleep infinity
```

Comment:
There's a lot of manual steps here just to set up the containers properly. Would it be possible to just move everything to a script?